q2_makarsa/_network.py

```diff
@@ -21,3 +21,18 @@ def _validate_(self, level):
 )
 
 Network = qiime2.plugin.SemanticType("Network")
+
+class NodeCommunityFormat(model.TextFileFormat):
+    def _validate_(self, level):
+        try:
+            table = pd.read_table(str(self))
+            if not isinstance(table, pd.DataFrame):
+                raise ValidationError("")
```

Comment:
please insert an error message here. Given that there is a separate `ParseError` I am guessing that this error would be caused by an empty table?